src/state/machines/expansion/warm_summer.ts

```diff
@@ -21,7 +21,7 @@ export const actions = {
   [`${cardPrefix}Action`]: assign(({ context }: { context: GameState }) =>
     produce(context, (draft) => {
       // Check if at least one measure has been implemented
-      const positiveEffect = context.policyMarket.table.some((policyCard) => policyCard.effect === "positive");
+      const positiveEffect = context.policyMarket.exhausted.length > 0;
```

Comment:
oh wow... that would work reliably. Many of policy cards just disappear when done.  I think we should have a special action `discardPolicyCard` and use it everywhere..

https://github.com/search?q=repo%3Ahelcomsecretariat%2Fecosfera-baltica-digital%20draft.policyMarket.active%20%3D%20without(&type=code

Reply:
I see that `cancelPolicyCard` actually put funding cards to exhausted... so yeah. It would work.

Should we refactor `Done` actions. What do you think?

Reply:
Yeah I think this is a good idea, but lets add a separate issue and work on it when we have more time for refactoring etc next week?